_posts/2013-05-02-install.markdown

```diff
@@ -186,26 +153,27 @@ Check your version of rubygems
 gem -v
 {% endhighlight %}
 
-Make sure it is equal or higher than `2.6.7`. Re-run the command that was failing previously.
-
-If you are still running into problems you can always find the latest version of rubygems online at [rubygems.org](https://rubygems.org/pages/download). If you click on **GEM** you will get the latest version.
-
-### During bundle install
+Make sure it is equal or higher than `2.6.7`.
 
-The `Gem::RemoteFetcher::FetchError: SSL_connect` can also occur during the `bundle install` stage when creating a new rails app.
+Download the `cacert.pem` file from
+[http://curl.haxx.se/ca/cacert.pem](http://curl.haxx.se/ca/cacert.pem).
```

Comment:
the browser certs should be up to date, `https` should be fine here - especially that `http` is redirected to `https`